gammapy/estimators/flux.py

```diff
@@ -58,50 +56,30 @@ class FluxEstimator(Estimator):
 
     def __init__(
         self,
-        source,
-        energy_min,
-        energy_max,
+        source=0,
         norm_min=0.2,
         norm_max=5,
         norm_n_values=11,
         norm_values=None,
         n_sigma=1,
-        n_sigma_ul=3,
+        n_sigma_ul=2,
         selection_optional=None,
         fit=None,
         # TODO: why the different default here?
-        reoptimize=True
+        reoptimize=False
```

Comment:
The docstring should be adapted accordingly.
I am wondering where we could explain in detail the details of this option, that is in fact quite import...

Reply:
Yes, I think we should try to avoid to repeat the documentation. Maybe a forward reference to the `Fit` class is a good option, where the `reoptimise` is explained in detail.